mpi-proxy-split/mpi-wrappers/mpi_collective_wrappers.cpp

```diff
@@ -170,27 +170,115 @@ USER_DEFINED_WRAPPER(int, Ibarrier, (MPI_Comm) comm, (MPI_Request *) request)
   return retval;
 }
 
+
+#if 0
+ /*
+ This version, MPI_Allreduce_reproducible, can be called from
+ the MPI_Allreduce wrapper and returned.  If desired, it could be
+ called selectively on certain sizes or certain types or certain op's.
+
+ Use MPI_Type_get_envelope and MPI_Type_get_contents
+   to discover if this is a dup of MPI_DOUBLE
```

Comment:
I don't understand the above code.  Is this PR valid only for `MPI_DOUBLE`, or other types also?  If it's only for `MPI_DOUBLE`, say that.